core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -30,18 +30,18 @@ oppia.directive('answerGroupEditor', [
           '&onSaveAnswerGroupCorrectnessLabel'),
         outcome: '=',
         suppressWarnings: '&',
-        rules: '='
+        rules: '=',
```

Comment:
why change this line?

Reply:
Oh, actually I added `trainingData` attribute but then removed it but forgot the comma.

Reply:
Done